client/src/components/layout/UserShow.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import UserBuildsList from "./UserBuildsList";
+import UserReviewsList from "./UserReviewsList"
+
+const UserShow = ({user}) => {
```

Comment:
space padding inside braces
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--in-braces